tests/validation/tests/v3_api/test_secrets.py

```diff
@@ -623,12 +623,33 @@ def create_project_client(request):
     namespace["project"] = p
     namespace["c_client"] = c_client
 
+
     def fin():
         client = get_user_client()
         client.delete(namespace["project"])
+
     request.addfinalizer(fin)
 
 
+def rancher_version_check():
+    # Checks if rancher version is greater than v2.6
+    # eg: v2.5-head, v2.5 etc.
+
+    rancher_version = get_setting_value_by_name('server-version')
+
+    version_check = False
+    if rancher_version.startswith('v'):
+        if "head" in rancher_version:
+            rancher_version = ''.join(rancher_version.split("-")[0])
+        else:
+            rancher_version = '.'.join(rancher_version.split(".")[:2])
+
+    if version.parse(rancher_version) > version.parse('v2.5'):
+        version_check = True
+
+    return version_check
```

Comment:
We should probably just return whatever the version comparison results to:
```
return version.parse(rancher_version) > version.parse('v2.5')
```

Reply:
fix should return True for any patch versions if they are 2.6 and above eg., v2.6.0, v2.6.1 etc and should return False for any patch versions of 2.5 eg., v2.5.0, v2.5.1,v2.5.9 etc

Fix to change the parameter `environment` to `env` was introduced from v2.6.0 and above. 

Adding v2.5.10 will result in error if the version is v2.5.11 as it returns true and the expected result be False.